RELEASE.md

```diff
@@ -65,8 +65,8 @@ Following is the release cadence for year 2023/2024. All dates below are tentati
 | --- | --- | --- | --- | --- |
 | 2.1 | Aug 2023 | Oct 2023 | Nov 2023 | Dec 2023 |
 | 2.2 | Dec 2023 | Jan 2024 | Feb 2024 | Mar 2024 |
-| 2.3 | Mar 2024 | Apr 2024 | May 2024 | Jun 2024 |
-| 2.4 | May 2024 | Jul 2024 | Aug 2024 | Sep 2024 |
+| 2.3 | Mar 2024 | Apr 2024 | Jun 2024 | --- |
```

Comment:
Nit (not sure how `---` will render in the table and whether it's clear that it means that there would not be 2.3.2)
```suggestion
| 2.3 | Mar 2024 | Apr 2024 | Jun 2024 | Not planned |
```

Reply:
It renders as ``---`` However Explicit is better! Agree.